docs/content/dagster-plus/deployment/azure/acr-user-code.mdx

```diff
@@ -89,18 +89,26 @@ First, we'll need to generate a service principal for GitHub Actions to use to a
 az ad sp create-for-rbac --name "github-actions-acr" --role contributor --scopes /subscriptions/<your_azure_subscription_id>/resourceGroups/<your_resource_group>/providers/Microsoft.ContainerRegistry/registries/<your_acr_name>
 ```
 
-This command will output a JSON object with the service principal details. Make sure to save the `appId`, `password`, and `tenant` values - we'll use them in the next step.
+This command will output a JSON object with the service principal details. Make sure to save the `appId` and `password` values - we'll use them in the next step.
 
 ### Add secrets to your repository
 
 We'll add the service principal details as secrets in our repository. Go to your repository in GitHub, and navigate to `Settings` -> `Secrets`. Add the following secrets:
 
+- `DAGSTER_CLOUD_API_TOKEN`: An agent token, for more details see [Managing agent tokens](/dagster-plus/account/managing-user-agent-tokens#managing-agent-tokens).
 - `AZURE_CLIENT_ID`: The `appId` from the service principal JSON object.
 - `AZURE_CLIENT_SECRET`: The `password` from the service principal JSON object.
 
-### Update the workflow
+### Update the GitHub Actions workflow
 
-Finally, we'll update the workflow to use the service principal details. Open `.github/workflows/dagster-cloud-deploy.yml` in your repository, and uncomment the section on Azure Container Registry. It should look like this:
+For this step, open `.github/workflows/dagster-cloud-deploy.yml` in your repository with your preferred text editor to perform the changes below.
+
+In the `env` section of the workflow, update the following variables:
+
+- `DAGSTER_CLOUD_ORGANIZATION`: The name of your Dagster Cloud organization.
```

Comment:
I don't actually remember needing to do this but maybe I hard-coded it. Seems fine to include this in the section tho

Reply:
i think this can be derived by default from the token in some circumstances but i like explicit over magic in documentation - makes the user more confident to self-serve.

Reply:
Yep makes sense to me